From 17277915af703a4767de791916621d8f59aef516 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Wed, 10 Feb 2016 23:21:26 +0100
Subject: [PATCH] config.c: add missing <sys/ttydefaults.h> include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This include is needed to get the definition of CEOT, otherwise the
build fails with:

config.c: In function ‘vpnc_getline’:
config.c:145:25: error: ‘CEOT’ undeclared (first use in this function)
   if (llen == 0 && c == CEOT)
                         ^

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 config.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config.c b/config.c
index 11b363b..f47a534 100644
--- a/config.c
+++ b/config.c
@@ -31,6 +31,7 @@
 #include <sys/types.h>
 #include <sys/utsname.h>
 #include <sys/wait.h>
+#include <sys/ttydefaults.h>
 
 #include <gcrypt.h>
 
-- 
2.6.4

